packages/db/src/index.ts

```diff
@@ -19,6 +19,11 @@ export const createKysely = (env: Environment) =>
       }),
     }),
     plugins: [new CamelCasePlugin()],
+    // Uncomment this to log all queries, useful for debugging
+    // log(event) {
+    //   console.log(event.query.sql)
+    //   console.log(event.query.parameters)
+    // }
```

Comment:
Put this behind an environment variable maybe?

Reply:
See Hørluck commnt